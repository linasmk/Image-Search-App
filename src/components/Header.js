/* ===== App Dependencies ===== */
import React from "react";
/* ============ Code ============ */

const Header = () => (
  <header className="header">
    <div className="header__inner-wrapper">
      <h1 className="title">Image Search App</h1>
    </div>
  </header>
);
export default Header;
