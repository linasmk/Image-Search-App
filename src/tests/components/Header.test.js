/* ========= App Dependencies ============= */
import React from "react";
import { shallow } from "enzyme";
/* ========= Components ============= */
import Header from "@comp/Header";
/* ========= Code ============= */
let wrapper;

beforeEach(() => {
  wrapper = shallow(<Header />);
});
it("renders Header component correctly", () => {
  expect(wrapper).toMatchSnapshot();
});

it("has HTML header element", () => {
  expect(wrapper.find("header").length).toEqual(1);
});

it("has HTML h1 element", () => {
  expect(wrapper.find("h1").length).toEqual(1);
});
