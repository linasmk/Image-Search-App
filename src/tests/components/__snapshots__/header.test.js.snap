// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header component correctly 1`] = `
<header
  className="header"
>
  <div
    className="header__inner-wrapper"
  >
    <h1
      className="title"
    >
      Image Search App
    </h1>
  </div>
</header>
`;
